Makefile

```diff
@@ -14,10 +15,18 @@ PKG=github.com/hashicorp/vault-k8s/version
 LDFLAGS?="-X '$(PKG).Version=v$(VERSION)'"
 TESTARGS ?= '-test.v'
 
+# kind cluster name
+KIND_CLUSTER_NAME?=vault-k8s
+
+# kind k8s version
+KIND_K8S_VERSION?=v1.29.2
+
 VAULT_HELM_CHART_VERSION ?= 0.27.0
+# TODO: add support for testing against enterprise
 VAULT_HELM_FLAGS?=--repo https://helm.releases.hashicorp.com --version=$(VAULT_HELM_CHART_VERSION) \
 	--wait --timeout=5m \
 	--values=test/vault/dev.values.yaml \
+	--set server.image.tag=$(VAULT_VERSION) \
```

Comment:
Can we also set `injector.agentImage.tag` in the same way, to keep them in sync?

Reply:
Added in b1e6b67041a227bce11620d8d4c684c6d645047e